modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowCalendar.testcase

```diff
@@ -525,15 +525,6 @@ definition {
 		CalendarEvent.viewStatusViaDetails(status = "Pending");
 	}
 
-	@description = "Verify that the Calendar Event status changes when approved"
```

Comment:
[LRQA-68597](https://liferay.atlassian.net/browse/LRQA-68597) This is already covered in WorkflowCalendar#CanApproveCalendarEventOnSingleApprover